code/datums/components/storage/storage.dm

```diff
@@ -848,7 +848,7 @@
 		return
 	A.add_fingerprint(user)
 	remove_from_storage(I, get_turf(user))
-	if(!user.put_in_hands(I))
+	if(INVOKE_ASYNC(!user.put_in_hands(I)))
```

Comment:
Put all the code from this point onwards in its own proc, and INVOKE_ASYNC call that--checking the return of an INVOKE_ASYNC like this doesn't make too much sense in this context.

Reply:
I've already done this, you're looking at an older version.

Reply:
ah